.gitignore

```diff
@@ -188,6 +188,7 @@ build_ios
 /build_*
 .build_debug/*
 .build_release/*
+.build_profile/*
```

Comment:
it's useful for building RelWithDebInfo builds

Reply:
Wasn't this supposed to be separated from a fuser upstream PR? Or, have we decided to sneak it in?

Reply:
I have an upstream PR (https://github.com/pytorch/pytorch/pull/44399). It's approved and imported, although I'm not sure what the "imported" means, but I can't _merge_ the PR due to unrelated CI failures?

In the meantime, I have this change in multiple branches, but I'll try to clean it up so it will not show up in PRs (I'll remove it here as well)